### PR TITLE
FORTECH-114: Fixed Item's label position and color

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Changes to Item's label styles to correspond to GUI design
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

File: styles/variables-base.less [hunk 1]
```diff
@@ -332,11 +332,15 @@
 @agate-item-content-min-width:        60%;
 @agate-item-font-size:                30px;
 @agate-item-font-weight:              @agate-base-font-weight;
+@agate-item-haslabel-padding:         20px 0 20px 0;
```
Comment (stanca-pop-lgp) [inline styles/variables-base.less:335]: Simplify this by writing "20px 0".

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: styles/variables-silicon.less [hunk 2]
```diff
@@ -182,6 +182,10 @@
 @agate-item-label-font-size:  30px;
 @agate-item-side-outset:      30px;
 @agate-item-sub-text-size:    24px;
+@agate-item-itemcontent-font-size:    33px;
+@agate-item-itemcontent-line-height:  36px;
+@agate-item-label-font-size:          24px;
+@agate-item-label-line-height:        27px;
```
Comment (jeonghee27) [inline styles/variables-silicon.less:188]: image

Is there any reason you change line-height value?

File: styles/variables-silicon.less [hunk 3]
```diff
@@ -182,6 +182,10 @@
 @agate-item-label-font-size:  30px;
 @agate-item-side-outset:      30px;
 @agate-item-sub-text-size:    24px;
+@agate-item-itemcontent-font-size:    33px;
```
Comment (jeonghee27) [inline styles/variables-silicon.less:185]: Do we need defined this variable?
Is it not enough to define it already in agate-item-sizing(); ?

File: styles/variables-silicon.less (hunk 2 above)
Comment (adrian-cocoara-lgp) [reply styles/variables-silicon.less:188]: You're right. I just forgot to made the changes before pushing the commit. I changed them according to GUI. Thanks for noticing! 😅 
I also removed the `padding-top` property, I don't think it is necessary

File: styles/variables-silicon.less (hunk 3 above)
Comment (adrian-cocoara-lgp) [reply styles/variables-silicon.less:185]: Oh, ok. I deleted this variable.

File: styles/variables-base.less [hunk 4]
```diff
@@ -344,11 +344,14 @@
 @agate-item-content-min-width:        60%;
 @agate-item-font-size:                30px;
 @agate-item-font-weight:              @agate-base-font-weight;
+@agate-item-haslabel-padding:         21px 0 21px 0;
```
Comment (ybsung) [inline styles/variables-base.less:347]: We might omit the 3rd and 4th values
```
@agate-item-haslabel-padding:         21px 0;
```

File: Item/Item.module.less [hunk 5]
```diff
@@ -49,12 +49,15 @@
 		}
 
 		.itemContent {
+			line-height: @agate-item-itemcontent-line-height;
```
Comment (ybsung) [inline Item/Item.module.less:52]: It might be better to add one empty line between styles and selectors.

File: Item/Item.module.less [hunk 6]
```diff
@@ -49,12 +49,15 @@
...
 			.label   {
 				color: @agate-item-label-color;
 				font-size: @agate-item-label-font-size;
+				line-height: @agate-item-label-line-height;
 			}
 
 			&.hasLabel {
+				padding: @agate-item-haslabel-padding;
```
Comment (ybsung) [inline Item/Item.module.less:60]: Same above.

**Review by ybsung — APPROVED** [review]
LGTM

**Pull request merged**